# [-1, 1]
float32 forward_backward
# [-1, 1]
float32 left_right